.gitignore

```diff
@@ -12,6 +12,7 @@
 .editorconfig
 .gopath/
 .go-pkg-cache/
+.vscode/
```

Comment:
This should be in your ~/.gitignore_global (see https://github.com/docker/docker/pull/30060/files#diff-a084b794bc0759e7a6b77810e01874f2R2 )
https://help.github.com/articles/ignoring-files/#create-a-global-gitignore